journal/src/main/java/io/zeebe/journal/file/record/RecordData.java

```diff
@@ -17,35 +17,32 @@
 
 import org.agrona.DirectBuffer;
 
-public class JournalIndexedRecordImpl implements JournalIndexedRecord {
+public class RecordData {
 
   private final long index;
   private final long asqn;
   private final DirectBuffer data;
 
-  public JournalIndexedRecordImpl(final long index, final long asqn, final DirectBuffer data) {
+  public RecordData(final long index, final long asqn, final DirectBuffer data) {
     this.index = index;
     this.asqn = asqn;
     this.data = data;
   }
 
-  @Override
   public long index() {
     return index;
   }
 
-  @Override
   public long asqn() {
     return asqn;
   }
 
-  @Override
   public DirectBuffer data() {
     return data;
   }
 
   @Override
   public String toString() {
-    return "JournalIndexedRecordImpl{" + "index=" + index + ", asqn=" + asqn + '}';
+    return "RecordData{" + "index=" + index + ", asqn=" + asqn + ", data=" + data + '}';
```

Comment:
Lately I've been using the `PrintBufferUtil` from Agrona here to get the hexDump with a max length - I still don't know if that's the best thing in general (maybe security concerns? though outputting the raw bytes is the same I guess), but it was very useful when debugging.

Reply:
Should we output data? I would say `data` can be removed from `toString`.

Reply:
Probably makes sense to remove it for now.